jsonnet/jsonnetfile.lock.json

```diff
@@ -18,7 +18,7 @@
           "subdir": "contrib/mixin"
         }
       },
-      "version": "91a5089d17f786d244ce49704774a29faf4e4ed2",
+      "version": "c2937d78d2722d774f69dbf91a956f382d32f4d3",
```

Comment:
We should only update the components that need to be updated in order to fix the bugzilla. So only telemeter-client and openshift-state-metrics. To do so you can manually keep only the versions changes of these dependencies with `git add -p` and discard the other ones.